addons/website/static/src/js/utils.js

```diff
@@ -4,7 +4,16 @@ import { intersection } from "@web/core/utils/arrays";
 import { csrf_token, _t } from "@web/legacy/js/services/core";
 import { renderToElement } from "@web/core/utils/render";
 
-
+let _languages;
+/**
+ * 
+ */
+async function loadLanguages(rpc) {
+    if(!_languages) {
+       _languages = rpc({ route : "/website/get_languages" });
```

Comment:
Line is strangely indented. (Starts at column 7 it seems)

In JSON constants, we use no space before the colon and one space after. (E.g.: `key: value`)